linode/instance/schema_resource.go

```diff
@@ -29,6 +29,38 @@ You may need to switch to an explicit disk configuration.
 Take a look at the example here:
 https://www.terraform.io/docs/providers/linode/r/instance.html#linode-instance-with-explicit-configs-and-disks`
 
+func resourcePlacementGroup() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: "The ID of the Placement Group to assign this Linode to.",
+			},
+			"compliant_only": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
```

Comment:
I think this can be computed when not configured by the user?

Reply:
Or default to API's default value?

Reply:
Nice observation! I'm not 100% sure if it'd make sense to make this field computed given it is only specified in the request body for instance creation/assignment to bypass strictness requirements and isn't returned by the API.

Reply:
Hmm, I wonder if we could just inherit from state in the read flatten function, or maybe we could just disable diffs on this field altogether? I think the solution will need to be hacky one way or another given this value isn't stateful on the API 🤔